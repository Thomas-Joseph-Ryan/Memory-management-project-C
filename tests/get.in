set a 1 2 3
get a
get A
get 1b
set b a 4 5 6
get b
