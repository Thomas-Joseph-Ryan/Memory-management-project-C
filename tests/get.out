> ok

> [1 2 3]

> no such key

> invalid input

> ok

> [a 4 5 6]

> 
bye
